pkg/syncer/metadatasyncer.go

```diff
@@ -1191,15 +1189,18 @@ func initResizeReconciler(ctx context.Context, tkgClient clientset.Interface, su
 		log.Errorf("resize: could not get supervisor namespace in which Tanzu Kubernetes Grid was deployed. Resize reconciler is not running for err: %v", err)
 		return err
 	}
-	onceForVolumeResizeReconciler.Do(func() {
-		log.Infof("initResizeReconciler is triggered")
-		informerFactory := informers.NewSharedInformerFactory(tkgClient, resizeResyncPeriod)
-
-		rc := newResizeReconciler(tkgClient, supervisorClient, supervisorNamespace, resizeResyncPeriod, informerFactory,
-			workqueue.NewItemExponentialFailureRateLimiter(resizeRetryIntervalStart, resizeRetryIntervalMax),
-		)
-		informerFactory.Start(wait.NeverStop)
-		rc.Run(ctx, resizeWorkers)
-	})
+	stopCh := make(chan struct{})
```

Comment:
What testing have you done to make sure that the resize reconciler is working fine?

Reply:
Update the test.